packages/cashier/src/modules/deposit-fiat/components/deposit-fiat-iframe/deposit-fiat-iframe.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from 'react';
+import { Loading } from '@deriv/components';
+import { useDepositFiatAddress } from '@deriv/hooks';
+import { observer, useStore } from '@deriv/stores';
+import './deposit-fiat-iframe.scss';
+
+const DepositFiatIframe: React.FC = observer(() => {
+    const { ui } = useStore();
+    const { is_dark_mode_on } = ui;
+    const { data: iframe_url, isSuccess, resend } = useDepositFiatAddress();
+    const [show_loader, setShowLoader] = useState(true);
+
+    React.useEffect(() => {
+        resend();
+        setShowLoader(true);
+    }, [is_dark_mode_on]);
+
+    return (
+        <React.Fragment>
+            {show_loader && (
+                <Loading
+                    className='deposit-fiat-iframe__loader'
+                    data-testid='dt_deposit_fiat_iframe_loader'
+                    is_fullscreen={false}
+                />
+            )}
```

Comment:
```suggestion
            {is_loading && <Loading is_fullscreen={false} />}
```
@aum-deriv `Loading` component by default has test_id of `dt_initial_loader`, So no need to add test_id for it.